cicd/gitops/working-with-openshift-gitops.adoc

```diff
@@ -0,0 +1,21 @@
+[id="working-with-openshift-gitops"]
+= Working with OpenShift GitOps
+include::modules/common-attributes.adoc[]
+include::modules/gitops-document-attributes.adoc[]
+:context: working-with-openshift-gitops
+toc::[]
+
+//Concept Module
+include::modules/about-openshift-gitops.adoc[leveloffset=+1]
```

Comment:
Let's change the id to about-gitops.adoc
I would consider Red Hat OpenShift GitOps and OpenShift GitOps synonymous.